keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/ScalarDoubleTensor.java

```diff
@@ -597,6 +602,21 @@ public DoubleTensor floorInPlace() {
         return this;
     }
 
+    @Override
+    public DoubleTensor roundInPlace() {
+        double valueToRound = value;
+        if (value < 0. && value + 0.5 == (double) value.intValue()) {
+            // This is necessary to match Python behaviour
```

Comment:
This might be confusing for someone reading this. Why would we want to match Python?
`this is necessary to match ND4J and Python behaviour` might read better?